web/src/test/webapp/spec/routeManipulationSpec.js

```diff
@@ -0,0 +1,74 @@
+// because leaflet-src.js requires window, document etc. we need to provide these variables...
```

Comment:
No idea, but looks okay to me. We are kind of mocking away these dependencies which could lead to trouble but for now this should be sufficient. In long term we should try to move more parts into a better structured code ;( (more seperate from UI)